straxen/plugins/led_calibration.py

```diff
@@ -0,0 +1,114 @@
+'''
+Dear nT analyser, 
+if you want to complain please contact: chiara@physik.uzh.ch, gvolta@physik.uzh.ch
+'''
+
+import strax
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+import numba
+from numba import njit
+
+#export, __all__ = strax.exporter()
+# QUESTION: what does 'export, __all__ = strax.exporter()' do?
+# ANSWER: [fill in]
+
+@strax.takes_config(
+    strax.Option('LED_window',
+                 default=(125, 250),
+                 help="Window (samples) where we expect the signal in LED calibration"),
+    strax.Option('noise_window',
+                 default=(350, 475),
```

Comment:
I fear afterpulses may be a concern if the noise window follows the LED window. Have you checked any differences if you use before/after the LED window?

Reply:
Not yet but it is on my TODO list. I didn' t think about it untill yesterday when Shingo pointed out. I will check